src/Routes.js

```diff
@@ -2,7 +2,8 @@ import React from 'react';
 import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
 import ProductLists from './pages/ProductLists/ProductLists';
 import Login from './pages/Login/Login';
-import Main from './pages/Main/Main';
+
+import Join from './pages/Join/Join';
 import Nav from './components/Nav/Nav/Nav';
 import main from './pages/Main/Main';
 import productDetail from './pages/ProductDetail/ProductDetail';
```

Comment:
- 컴포넌트 이름은 대문자로 시작해주세요!
```suggestion
import Main from './pages/Main/Main';
import ProductDetail from './pages/ProductDetail/ProductDetail';
```